src/components/CalltoActionButton.js

```diff
@@ -1 +1,43 @@
-//Gets passed a prop LinkTo that leads to the desired section, exists in two different styles based on its props (https://emotion.sh/docs/styled)
\ No newline at end of file
+import styled from "@emotion/styled"
+import React from "react"
+
+const ButtonLarge = styled.button`
+  border: solid 3px transparent;
+  border-radius: 100rem;
+  background-image: linear-gradient(
+      rgba(255, 255, 255, 0),
+      rgba(255, 255, 255, 0)
+    ),
+    linear-gradient(101deg, #78e4ff, #ff48fa);
+  background-origin: border-box;
+  background-clip: content-box, border-box;
+  padding: 0.5rem 3rem;
+  box-shadow: 2px 1000px 1px #111 inset;
+  color: white;
+  font-family: inherit;
+
+  & :hover {
+    animation: pulse 3s;
+    box-shadow: 0 0 0 2em rgba(#fff, 0);
+  }
+
+  @keyframes pulse {
+    0% {
+      box-shadow: 0 0 0 0 var(--gradient);
+    }
+  }
```

Comment:
If you like to reuse keyframes (or outsource), take a look at https://emotion.sh/docs/keyframes

Reply:
Thanks, would love to talk about outsourcing: Is the idea to make the keyframes reusable throughout the whole project? Wouldn't we then move away from component scoped css?

Reply:
Think about an animation/keyframes module (e.g. in `src/utils/keyframes.js`) which exports different animations you use in your app. You might do the same for icons or other images, which are stored in different folders. 
You wouldn't move away from component scoped css in my eyes.